src/btree/bt_vrfy.c

```diff
@@ -611,19 +721,26 @@ __verify_tree(
         break;
     }
 
+    if (vs->dump_tree_shape)
+        printf("\n");
+
     /* Check tree connections and recursively descend the tree. */
     switch (page->type) {
     case WT_PAGE_COL_INT:
         /* For each entry in an internal page, verify the subtree. */
         entry = 0;
+        next_stack_level = WT_MIN(vs->depth + 1, WT_ELEMENTS(vs->depth_internal) - 1);
         WT_INTL_FOREACH_BEGIN (session, page, child_ref) {
+            vs->tree_stack[my_stack_level] = entry++;
+            vs->cells_count_stack[next_stack_level] = vs->total_sz_stack[next_stack_level] =
+              vs->key_sz_stack[next_stack_level] = vs->val_sz_stack[next_stack_level] = 0;
+
             /*
              * It's a depth-first traversal: this entry's starting record number should be 1 more
              * than the total records reviewed to this point. However, for VLCS fast-truncate can
              * introduce gaps; allow a gap but not overlapping ranges. For FLCS, gaps are not
              * permitted.
              */
-            ++entry;
```

Comment:
Is it correct to remove this?

Reply:
It's moved over to the `vs->tree_stack[my_stack_level] = entry++;` line